cuegui/cuegui/JobMonitorTree.py

```diff
@@ -384,7 +384,8 @@ def _getUpdate(self):
                         monitored_proxies.remove(proxy)
 
             if monitored_proxies:
-                for job in opencue.api.getJobs(id=monitored_proxies, all=True):
+                for job in opencue.api.getJobs(id=[i.split('.')[-1] for i in monitored_proxies],
```

Comment:
nit: generally prefer meaningful var names instead of `i`, even in a comprehension like this.